R/infrastructure-git.R

```diff
@@ -190,10 +241,12 @@ use_git_ignore <- function(ignores, directory = ".", pkg = ".") {
 #' those fields already exist.
 #'
 #' @inheritParams use_git
+#' @param host GitHub host to use. Override with your GitHub enterprise
+#'    hostname, for example, "https://github.hostname.com".
 #' @family git infrastructure
 #' @keywords internal
 #' @export
-use_github_links <- function(pkg = ".") {
+use_github_links <- function(pkg = ".", host = "https://github.com") {
```

Comment:
I'm happy to make the change for `github_URL`, but I don't think `issues_URL` is returning what we want. The hostname in `issues_URL` is `api.github.com`, where we want `github.com`

As a first approach, could I just append `"/issues"` to `github_URL`?